src/SerialFirmata.h

```diff
@@ -142,6 +146,12 @@ class SerialFirmata: public FirmataFeature
     int serialBytesToRead[SERIAL_READ_ARR_LEN];
     signed char serialIndex;
 
+#if defined(SERIAL_STORE_AND_FORWARD)    
+    unsigned long lastReceive[SERIAL_READ_ARR_LEN];
+    unsigned char maxCharDelay[SERIAL_READ_ARR_LEN];
+    int lastAvailableBytes[SERIAL_READ_ARR_LEN];
```

Comment:
Thx, good to know that it also works in different setups.

To reduce the data size for variables if store&forward is not used I suggest to keep the define in the header file and add another around line 337.